examples/koa/package.json

```diff
@@ -32,15 +32,15 @@
   },
   "dependencies": {
     "@koa/router": "^9.3.1",
-    "@opentelemetry/api": "^1.0.0-rc.0",
-    "@opentelemetry/exporter-jaeger": "^0.19.0",
-    "@opentelemetry/exporter-zipkin": "^0.19.0",
-    "@opentelemetry/instrumentation": "^0.19.0",
-    "@opentelemetry/instrumentation-http": "^0.19.0",
+    "@opentelemetry/api": "^0.20.0",
+    "@opentelemetry/exporter-jaeger": "^0.20.0",
+    "@opentelemetry/exporter-zipkin": "^0.20.0",
+    "@opentelemetry/instrumentation": "^0.20.0",
+    "@opentelemetry/instrumentation-http": "^0.20.0",
     "@opentelemetry/instrumentation-koa": "^0.15.0",
-    "@opentelemetry/node": "^0.19.0",
-    "@opentelemetry/tracing": "^0.19.0",
-    "axios": "^0.19.0",
+    "@opentelemetry/node": "^0.20.0",
+    "@opentelemetry/tracing": "^0.20.0",
+    "axios": "^0.20.0",
```

Comment:
```suggestion
    "axios": "^0.21.1",
```

Reply:
are we really using a 0.x axios dep?

Reply:
according to npm 0.21.1 is the latest version.